tests/test_freetextsearch.py

```diff
@@ -1,12 +1,1325 @@
+                   'AFF2',
+                   'AFF3',
+                   'AFF4',
+                   'AFG3L2',
+                   'AFM',
+                   'AFMID',
+                   'AFP',
+                   'AFTPH',
+                   'AGA',
+                   'AGAP1',
+                   'AGAP2',
+                   'AGAP3',
+                   'AGAP4',
+                   'AGAP5',
+                   'AGAP6',
+                   'AGAP9',
+                   'AGBL1',
+                   'AGBL2',
+                   'AGBL3',
+                   'AGBL4',
+                   'AGBL5',
+                   'AGER',
+                   'AGFG1',
+                   'AGFG2',
+                   'AGGF1',
+                   'AGK',
+                   'AGL',
+                   'AGMAT',
+                   'AGMO',
+                   'AGO1',
+                   'AGO2',
+                   'AGO3',
+                   'AGO4',
+                   'AGPAT1',
+                   'AGPAT2',
+                   'AGPAT3',
+                   'AGPAT4',
+                   'AGPAT5',
+                   'AGPS',
+                   'AGR2',
+                   'AGR3',
+                   'AGRN',
+                   'AGRP',
+                   'AGT',
+                   'AGTPBP1',
+                   'AGTR1',
+                   'AGTR2',
+                   'AGTRAP',
+                   'AGXT',
+                   'AGXT2',
+                   'AHCTF1',
+                   'AHCY',
+                   'AHCYL1',
+                   'AHCYL2',
+                   'AHDC1',
+                   'AHI1',
+                   'AHNAK',
+                   'AHNAK2',
+                   'AHR',
+                   'AHRR',
+                   'AHSA1',
+                   'AHSA2',
+                   'AHSG',
+                   'AHSP',
+                   'AICDA',
+                   'AIDA',
+                   'AIF1',
+                   'AIF1L',
+                   'AIFM1',
+                   'AIFM2',
+                   'AIFM3',
+                   'AIG1',
+                   'AIM1',
+                   'AIM1L',
+                   'AIM2',
+                   'AIMP1',
+                   'AIMP2',
+                   'AIP',
+                   'AIPL1',
+                   'AIRE',
+                   'AJAP1',
+                   'AJUBA',
+                   'AK1',
+                   'AK2',
+                   'AK3',
+                   'AK4',
+                   'AK5',
+                   'AK6',
+                   'AK7',
+                   'AK8',
+                   'AK9',
+                   'AKAIN1',
+                   'AKAP1',
+                   'AKAP2',
+                   'AKAP3',
+                   'AKAP4',
+                   'AKAP5',
+                   'AKAP6',
+                   'AKAP7',
+                   'AKAP8',
+                   'AKAP8L',
+                   'AKAP9']
 
-from app import create_app
-from tests import GenericTestCase
 
```

Comment:
Just curious - why do you have them all on separate line? Or is that just because you copy pasted from the file?
I can give you the same without newlines too if you are interested